client/src/service/game/game.ts

```diff
@@ -13,21 +14,28 @@ export function initialiseGame(
   const players: Player[] = [];
 
   for (let i = 1; i < numberPlayers + 1; i++) {
-    const cards = getCardsForPlayer(numberOfRoundsToPlay);
+    const cards: Card[] =
+      i === 1
+        ? DUMMY_CARD_DATA1
+        : i === 2
+        ? DUMMY_CARD_DATA2
+        : getCardsForPlayer(numberOfRoundsToPlay);
```

Comment:
I don't think we want the dummy data here? I assume it was for testing your work? If so before merging to main I suggest reverting this change. The game.test file is failing as it's receiving 2 not 4 cards as the test is expecting.

Reply:
Not just for testing, the rounds don't run without cards to run them with, so I needed the dummy data to actually get it working at all. I can add another three cards to each hand so the game.test file gets the cards it's expecting? There wasn't any card data I could see elsewhere though, where is the game test getting the four cards from?

The dummy data will all be removed once we have the real data anyway.

Reply:
On `initialiseGame` the number of rounds is a parameter. That's passed to `getCardsForPlayer` in `game.ts` and a for loop is used to create the cards with fake data.

Reply:
Ah! Yes sorry I remember the specific issue was that the dummy cards generated by getCardsForPlayer don't have any stats data, which I needed for the rounds logic to work. I already had the fake cards I'd written with stats, that's why I used those.

If you can add something getCardsForPlayer to add stats data to the cards I can remove this and it should be fine? Alternatively we can remove both once we have real data anyway.

The cards are also now typed so that they can't have an empty stats array anyway.

Reply:
So basically either, getCardsForPlayer needs to also create stats data for the cards, or I need to add three extra cards to my dummy data until we have the real data to work with. Let me know which you'd prefer?

Reply:
I don't mind - which ever is easier to get the PR in. as we'll be able to replace it soon.